src/Nexus/Extensibility/DataSource/DataSourceController.cs

```diff
@@ -130,16 +142,29 @@ internal class DataSourceController(
         if (catalogRegistrations.Any(catalogRegistration => !catalogRegistration.Path.StartsWith(path)))
```

Comment:
[Stackoverflow](https://stackoverflow.com/a/879533) told me that `.Exists` (which I did not knew before) is a legacy method only applicable to `List<T>` instead of the more general `IEnumerable<T>` and not directly compatible with LINQ. For the sake of consistency I would prefer staying with `Any`.

Reply:
Okay, than we keep Any. :-)